packages/php-wasm/universal/src/lib/supported-php-extensions.ts

```diff
@@ -0,0 +1,4 @@
+const supportedPHPExtensions = ['mbstring', 'xml-bundle', 'gd'] as const;
```

Comment:
I'm not super happy about inventing a name like `xml-bundle`, but I also don't want to make it super granular like `libxml`, `xml`, `dom`, `simplexml`, `xmlreader`, `xmlwriter`. What do you think @dmsnell?

Reply:
> I also don't want to make it super granular 

why not? this is how PHP itself does it isn't it?

there's more impetus here to minimize downloaded bytes, so granularity seems fine.

what could be a good compromise is making a few common bundles, including some kind of "WP minimum" and "WP kitchen sink" that cover what most common WordPress installs have with the versions of PHP built in the common Linux distros/macOS/Windows.

perhaps it would be clearer if there were _extensions_ and _extensionBundles_ or _basicEnvironments_ that could be independently added. we wouldn't have to worry about the name clash between the two concepts.

Reply:
I was concerned with the developer experience of having to specify a ton of options. Extension bundles would solve that, though, thank you! I'm updating this PR now